samples/ingestion/ingestion-client/StartTranscriptionByServiceBus/Program.cs

```diff
@@ -0,0 +1,51 @@
+// <copyright file="Program.cs" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE.md file in the project root for full license information.
+// </copyright>
+
+namespace StartTranscription
+{
+    using System.Linq;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Functions.Worker;
+    using Microsoft.Extensions.DependencyInjection;
+    using Microsoft.Extensions.Hosting;
+    using Microsoft.Extensions.Logging;
+
+    /// <summary>
+    /// Represents the entry point of the application.
+    /// </summary>
+    public static class Program
+    {
+        /// <summary>
+        /// Main entry point of the function app.
+        /// </summary>
+        /// <param name="args"></param>
+        /// <returns>A <see cref="Task"/> representing the result of the asynchronous operation.</returns>
+        public static async Task Main(string[] args)
+        {
+            var host = new HostBuilder()
+                .ConfigureFunctionsWorkerDefaults()
+                .ConfigureServices(s =>
+                {
+                    s.AddApplicationInsightsTelemetryWorkerService();
+                    s.ConfigureFunctionsApplicationInsights();
+
+                    s.Configure<LoggerFilterOptions>(options =>
+                    {
+                        // The Application Insights SDK adds a default logging filter that instructs ILogger to capture only Warning and more severe logs. Application Insights requires an explicit override.
+                        // Log levels can also be configured using appsettings.json. For more information, see https://learn.microsoft.com/en-us/azure/azure-monitor/app/worker-service#ilogger-logs
+                        var toRemove = options.Rules.FirstOrDefault(rule => rule.ProviderName
+                            == "Microsoft.Extensions.Logging.ApplicationInsights.ApplicationInsightsLoggerProvider");
+                        if (toRemove is not null)
+                        {
+                            options.Rules.Remove(toRemove);
+                        }
+                    });
```

Comment:
nit: since this is mostly duplication and I don't think we'd like to set up different logging for different functions we could consider sharing the code between funcs

Reply:
Pulled this into an extension method `ConfigureIngestionClientLogging`